editor/src/document/dialogs/about_dialog.rs

```diff
@@ -0,0 +1,45 @@
+use crate::{layout::widgets::*, message_prelude::FrontendMessage};
+
+pub struct AboutGraphite {
+	pub release: String,
+	pub timestamp: String,
+	pub hash: String,
+	pub branch: String,
+}
```

Comment:
@0HyperCube What do you think about storing the `release`, `timestamp`, `hash`, and `branch` as metadata properties on the entire Editor instance, rather than on a struct that's specific to the About Graphite Dialog? That way we can read the value in other parts of the app. And then the About Graphite Dialog is one of the readers of that instance global metadata. So basically make this `AboutGraphite` struct have no values, but define another struct for build metadata that would live on something like the dispatcher or global message handler or somewhere else that'd be appropriate. Thoughts?

Reply:
My previous commit stores the `AboutGraphite` struct publicly in the `PortfolioMessageHandler`.

Reply:
What I meant to describe was where it's defined, which is in `dialogs/about_graphite.rs`. It seems to me that decoupling the concept of that metadata from its About Graphite dialog user would be cleaner. So defining that struct, perhaps called something else like `BuildInfo`, elsewhere might be nicer? Open to your thoughts though!

Reply:
What other things need to have access to the build metadata?

Reply:
Currently nothing, so maybe this is a good case of [YANGI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it). But it sort of just seems cleaner to separate those details, although I respect if you disagree with my opinion on that.

Reply:
You mean have another struct that contains the details and then have the dialog get a clone of that struct?

Reply:
That's what I had in mind, yes, because it seems to be a cleaner separation of concerns. Perhaps that's just my skewed perspective though.